app/views/shared/_item.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="col-md-4">
+  <div class="item-display">
```

Comment:
I don't think the divs need to have individual id's as they will be styled in the same way